scapy/layers/bluetooth.py

```diff
@@ -8,14 +8,14 @@
 Bluetooth layers, sockets and send/receive functions.
 """
 
+import ctypes
 import socket
 import struct
 import array
-from ctypes import *
```

Comment:
This is some cleanup: previously importing scapy ==> import all types from ctypes.